frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -243,6 +249,31 @@ const InfoControl: React.FC<InfoControlProps> = ({ open, setOpen, onHeaderAction
             id="slideOutInfoButton"
             variant="outline-secondary"
             onClick={() => {
+              const propertyTypeId = popUpContext.propertyTypeID;
```

Comment:
Not sure if the `popUpContext.propertyTypeID` is correct case on `ID`.  Might be fine.

Reply:
it is in this case (had it the other way originally and got a compile warning).